src/components/nav.html

```diff
@@ -0,0 +1,23 @@
+
+<div class="header-navbar">
```

Comment:
u should using semantic html . pls check [this](https://www.w3schools.com/html/html5_semantic_elements.asp)